docs/docs/guides/sso.mdx

```diff
@@ -79,6 +79,10 @@ For a production Infrahub instance at `infrahub.example.com`:
   </TabItem>
 </Tabs>
 
+:::note Reverse proxy setup
+Use the `INFRAHUB_PUBLIC_URL` environment variable to specify the externally accessible URL for your Infrahub instance. This configuration is critical when Infrahub is deployed behind a reverse proxy, load balancer, or in containerized environments where the internal and external URLs differ.
```

Comment:
Perhaps mention that this URL should match the one that's used within the SSO provider.